__init__.py

```diff
@@ -208,6 +193,31 @@ def handle_light_decrease_intent(self, message):
         message.data["Action"] = "down"
         self._handle_light_adjust(message)
 
+    @intent_handler('automation.intent')
+    def handle_automation_intent(self, message):
+        self.log.debug("Automation trigger intent on entity: "+message.data.get("entity"))
+        message.data["Entity"] = message.data.get("entity")
```

Comment:
Is there a reason to reassign these to a capitalized key instead of using the lowercase key in the handler functions?

Reply:
Yes, reassign was introduced in one of previous PR by one of my predecessor as workaround to prevent loosing data from Padatious core as I remember, maybe I can find a first commit with it.

Reply:
I'm not really a fan of this, I think it's a good candidate for a follow up refactor but not going to block merging on it.